src/pve_exporter/collector.py

```diff
@@ -244,6 +244,51 @@ def collect(self): # pylint: disable=missing-docstring
 
         return itertools.chain(metrics.values(), info_metrics.values())
 
+class ClusterNodeConfigCollector(object):
+    """
+    Collects Proxmox VE VM information directly from config, i.e. boot, name, onboot, etc.
+    For manual test: "pvesh get /nodes/<node>/<type>/<vmid>/config"
+
+    # HELP pve_vm_config_onboot Proxmox vm config onboot value
+    # TYPE pve_vm_config_onboot gauge
+    pve_vm_config_onboot{id="qemu/113",node="XXXX",type="qemu"} 1.0
+    """
+
+    def __init__(self, pve):
+        self._pve = pve
+
+    def collect(self): # pylint: disable=missing-docstring
+        metrics = {
+            'onboot': GaugeMetricFamily(
+                'pve_vm_config_onboot',
+                'Proxmox vm config onboot value',
+                labels=['id', 'node', 'type']),
+            'memory': GaugeMetricFamily(
+                'pve_vm_config_memory',
+                'Proxmox vm config memory value',
+                labels=['id', 'node', 'type']),
```

Comment:
There is `pve_memory_size_bytes` already, so this metric seems duplicate. Do you have a case where the new metric is better than the existing `pve_memory_size_bytes`?

Reply:
I'll remove it seems like duplicate. I wanted to test it with ballooning memory management if the API cluster/resources does not return current value of ballooning, but don't have time for that ATM. Maybe in the futre will sent another PR for this if i'll test that.